packages/synapse-react-client/src/components/Authentication/PasswordField.tsx

```diff
@@ -0,0 +1,17 @@
+import TextField from '../TextField'
+import React from 'react'
+import { TextFieldProps } from '@mui/material'
+
+export default function PasswordField(props: TextFieldProps) {
```

Comment:
Extracted this Password field for reuse in the custom form on the Reset 2FA page